py-polars/polars/expr/expr.py

```diff
@@ -681,9 +681,9 @@ def alias(self, name: str) -> Expr:
 
         See Also
         --------
-        map
-        prefix
-        suffix
+        name.map
+        name.prefix
+        name.suffix
```

Comment:
Given these are no longer in the same namespace as `alias`, this needs to be done for them to link correctly in the docs.